src/helpers/offlineSyncModal.js

```diff
@@ -83,15 +92,17 @@ function offlineSyncModalFactory(proxy) {
             dialogShortcut.clear();
         })
         .on('wait', () => {
+
+            hider.show('.between-buttons-text');
```

Comment:
What are you targeting here? what is the scope?
it looks like a global page element